v2/README.md

```diff
@@ -26,13 +22,37 @@ For everything to work as expected make sure of the following:
 
 ## Usage
 
-Edit `.env` for your development requirements - you might need to point to local services running in an IDE for example.
+Please follow the instructions in [stacks README](./stacks/README.md) to run each stack
+
+## Code structure
+
+The required configs and scripts have been structured as follows:
+
+### dockerfiles
+
+Contains `Dockerfile.dp-compose` files for services that do not have a `Dockerfile.local` yet. Each repository should have its own `Dockerfile.local`, so this `dockerfiles` folder can be removed when this is the case.
+
+### manifests
+
+Contains docker compose config `yml` files for each service that is required by any of the stacks. These configurations are stack agnostic and define all the necessary env vars to run the services in any possible configuration that might be required by any stack. Each env var has a sensible default value, which will be used if not provided by the stack, and usually corresponds to the default value in the service config.
+
+The files are organised in subfolders according to their type:
+- core-ons: Core services implemented by ONS
```

Comment:
Please insert a blank line between lines 39 and 40 to avoid a markdown linter error, which requires lists to be surrounded by blank lines. And also between lines 46 and 47 below.

Reply:
👍